docs/workgroup/charter.rst

```diff
@@ -0,0 +1,136 @@
+===================================
+CPython Documentation Working Group
+===================================
+
+Purpose & Common Goals
+----------------------
+
+This workgroup will support efforts to improve
+and maintain CPython's documentation through curation,
+internationalization, and special projects. Its focus will also include
+tutorials, how-to documents, the devguide and other documentation in support
+of the PSF's mission to expand the global Python userbase.
+Quality and effectiveness will be key goals as we build a
+diverse team of community members, documentarians, and core developers.
+
+Things the workgroup could do:
+
+- Develop the governance model for docs
+- Be the steering council for docs
+- Be the authors of doc guidelines
+- Be the authors/editors of docs
+- Request grant funding from PSF, if needed
+- Form an editorial board for docs including core devs, educators, and
+  documentarians in the Python community
+
+Active Time
+-----------
+
+This workgroup will be active indefinitely while supported by the Python
+Steering Council.
+
+Core Values & Internal Governance
+---------------------------------
+
+The core values for this group are:
+
+- Respect
+- Support
+- Positive Atmosphere
+- Constructiveness
+- Integrity
+
+The workgroup adopts the `PSF Code of Conduct <https://www.python.org/psf/codeofconduct/>`_.
+Any action by a workgroup member, as decided by a majority of the group,
+that violates the principles in the Code of Conduct will result in that member
+being removed from the workgroup.
+
+Rules & Guidelines
+------------------
+
+- At any time, the Python Steering Council may, at its discretion, add any number
+  of non-voting observers to the working group to observe and participate in
+  discussion.
+- The group's purview includes the Sphinx documentation included in the Python
+  source repository, including the doc theme, as well as separate projects such
+  as the developer guide. Translations and infrastructure will be managed by
+  Julien Palard or a Steering Council appointed member.
+- The group will also maintain documentation of meetings and best practices.
+- The editorial board will include people outside of the core developers who are
+  tech writers or educators.
+
+Decision-Making Procedures
+--------------------------
+
+Decisions will be made using the voting procedures outlined below. Anyone may
+propose or request a topic for a decision.
+
+In general, a topic for decision should address one of the workgroup's goals:
+
+- Improve or maintain the state of Python's documentation effort by some
+  noticeable or measurable amount.
+- Increase the participation of contributors to documentation.
+
+To be considered for membership, prospective members must send an
+email introducing themselves, along with a description of why they want to be
```

Comment:
An email to whom?

Reply:
No idea. This needs to be updated, but not within my purview as a copyeditor.

Reply:
After this gets merged I'll capture all of the problems I found that are broader than strict copy-editing so that they dion't get lost.

A

Reply:
Good point, though this actually is mostly captured by #3